pkg/components/internal/testutil/jsonpath.go

```diff
@@ -104,3 +102,16 @@ func MatchJSONPathInt64Value(t *testing.T, yamlConfig string, jsonPath string, e
 		t.Fatalf("Expected: %d, Got: %d", expected, got)
 	}
 }
+
+// JSONPathExists checks if the given YAML config has an object at the given JSON path, also provide
+// what error to expect.
+func JSONPathExists(t *testing.T, yamlConfig string, jsonPath string, errExp string) {
```

Comment:
can this and other helper functions of JSONPath, such as Int64 helper, string helper be unified ?

Reply:
This function does not care about the value of the JSON path. It is only concerned with if the specified path exists.